static/app/views/prevent/tests/onboardingSteps/addScriptToYamlStep.tsx

```diff
@@ -10,23 +8,20 @@ interface AddScriptToYamlStepProps {
   step: string;
 }
 
-const SNIPPET = `- name: Upload test results to Sentry Prevent
+const SNIPPET = `
+- name: Upload test results to Sentry Prevent
   if: \${{ !cancelled() }}
   uses: getsentry/prevent-action@v0
   with:
     token: \${{ secrets.SENTRY_PREVENT_TOKEN }}
-`;
+`.trim();
 
 export function AddScriptToYamlStep({step}: AddScriptToYamlStepProps) {
   const headerText = tct(
-    'Step [step]: Add the script [actionName] to your CI YAML file',
+    'Step [step]: Add the workflow action [code:getsentry/prevent] to your CI YAML file',
     {
       step,
-      actionName: (
-        <Text size="xl" variant="promotion">
-          {t('permissions')}
-        </Text>
```

Comment:
Swapped out "promotion" text highlighting for using a semantic `code` element. If we really want it to be purple we can adjust the usage of `code` in this block

Reply:
Talked in slack, we'll adjust this globally for sentry.